assets/helpers/payPalContributionsCheckout/payPalContributionsCheckout.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import { toGuCountryCode } from 'helpers/internationalisation/country';
+
+
+// ----- Types ----- //
+
+type PayPalPostData = {
```

Comment:
There is another definition of PayPalPostData in `PayPalContributionsCheckoutActions`

Reply:
This file was created from that file. That file is going to be deleted.